tests/test_loaders.py

```diff
@@ -233,6 +233,8 @@ def test_qvalue_filtering(self):
         )
         self.assertEqual(obj.rawinput.shape, (2071, 10))
 
+    # TODO figure out what this is about or delete
```

Comment:
https://github.com/MannLabs/alphapeptstats/blob/91849c6b03bb0ff19f05bbc609dc5348b99c2ad4/alphastats/loader/SpectronautLoader.py#L82

This is raised when no EG.Qvalue is present in the file.

Reply:
ah, thanks, I could fix the test